packages/salesforcedx-vscode-apex/src/views/testDataAccessObjects.ts

```diff
@@ -35,20 +35,21 @@ export class TestSummarizer {
     group: ApexTestGroupNode
```

Comment:
`group` parameter is now unused, let's remove it.

Reply:
Okay